### PR TITLE
chore: Add copy feature buttons next to names

### PR DESCRIPTION
**Pull request by kyle-ssg**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

image

Uploading copy.mov…

## How did you test this code?

As above

### CODE REVIEW

**Comment by dabeeeenster** [comment]
LGTM

**Comment by matthewelwell** [comment]
@kyle-ssg can we see a screenshot of this with some segment overrides, tags, etc. ? That's why we didn't implement this the first time, and went with the additional actions button.

**Comment by kyle-ssg** [comment]
Sorry yep I thought the video had finished uploading, can see ours here on the vercel preview

image

image

https://github.com/Flagsmith/flagsmith/assets/8608314/dddbdae6-d60a-4396-b378-80385882959d

**Comment by dabeeeenster** [comment]
This looks great

File: frontend/web/styles/project/_buttons.scss [hunk 1]
```diff
@@ -129,7 +129,28 @@ button.btn {
       box-shadow: none;
     }
   }
-
+  &.btn-icon {
+    padding:0;
```
Comment (zachaysan) [inline frontend/web/styles/project/_buttons.scss:133]: tiny nit
```suggestion
    padding: 0;
```

**Review by zachaysan — APPROVED** [review]
Looks great! Only one tiny nit.

**Pull request merged**